client/src/redux/prop-types.ts

```diff
@@ -347,3 +352,53 @@ export interface UserFetchState {
   errored: boolean;
   error: string | null;
 }
+
+// Exam Related Types:
+interface GeneratedExamAnswer {
+  id: string;
+  answer: string;
```

Comment:
I noticed. It won't be that for this generated exam, It will be `question.answers.answer` - which isn't great etiher, but it's a little better. It will be `answer.answer` for the user exam. Perhaps just change the second answer to `text`.

Reply:
`text` seems fine to me.